native/src/components/TtsPlayer.tsx

```diff
@@ -0,0 +1,344 @@
+import { useTranslation } from 'react-i18next'
+import { AppState } from 'react-native'
+import Tts from 'react-native-tts'
+import styled from 'styled-components/native'
+
+import { CloseIcon, NoSoundIcon, PauseIcon, PlaybackIcon, PlayIcon, SoundIcon } from '../assets'
+import { isRTL } from '../constants/contentDirection'
+import { AppContext } from '../contexts/AppContextProvider'
+import { extractSentencesFromHtml } from '../utils/TtsPlayerUtils'
+import { reportError } from '../utils/sentry'
+import Slider from './Slider'
+import Icon from './base/Icon'
+import IconButton from './base/IconButton'
+import Text from './base/Text'
+
+const StyledTtsPlayer = styled.View<{ $isPlaying: boolean }>`
+  background-color: ${props => props.theme.colors.grayBackgroundColor};
+  border-radius: 28px;
+  width: 85%;
+  display: flex;
+  flex-direction: ${props => (props.$isPlaying ? 'column' : 'row')};
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+  padding: 6px;
+  position: absolute;
+  bottom: 5px;
+  min-height: 93px;
+  gap: ${props => (props.$isPlaying ? '5px;' : '10px')};
+`
+
+const StyledPanel = styled.View`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 20px;
+`
+
+const StyledPlayIcon = styled(IconButton)`
+  background-color: ${props => props.theme.colors.textColor};
+  width: 50px;
+  height: 50px;
+  border-radius: 50px;
+`
+
+const StyledBackForthButton = styled.TouchableOpacity`
+  display: flex;
+  flex-direction: row;
+  gap: 5px;
+  align-items: flex-end;
+`
+
+const PlayButtonIcon = styled(Icon)`
+  color: ${props => props.theme.colors.grayBackgroundColor};
+`
+
+const StyledNoSoundIcon = styled(Icon)`
+  height: 18px;
+  width: 18px;
+`
+
+const FlipView = styled.View`
+  transform: scaleX(-1);
+`
+
+const StyledText = styled(Text)`
+  font-weight: bold;
+`
+
+const StyledPlayerHeaderText = styled(Text)`
+  font-weight: 600;
+  align-self: center;
+  font-size: 18px;
+`
+
+const CloseButton = styled.TouchableOpacity`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  border-radius: 7px;
+  background-color: ${props => props.theme.colors.themeColor};
+  padding: 5px;
+  gap: 5px;
+  width: 176px;
+`
+
+const CloseView = styled.View`
+  flex-direction: column;
+  gap: 10px;
+`
+
+export type ttsContextType = {
+  content: string | null
+  setContent: React.Dispatch<React.SetStateAction<string | null>>
+  sentenceIndex: number
+  setSentenceIndex: React.Dispatch<React.SetStateAction<number>>
+  visible: boolean
+  setVisible: React.Dispatch<React.SetStateAction<boolean>>
+  title: string
+  setTitle: React.Dispatch<React.SetStateAction<string>>
+  volume: number
+  setVolume: React.Dispatch<React.SetStateAction<number>>
+}
+
+/* eslint-disable @typescript-eslint/no-empty-function */
+export const ttsContext = createContext<ttsContextType>({
+  content: null,
+  setContent: () => {},
```

Comment:
```suggestion
  setContent: () => undefined,
```
for all the empty functions and remove es-lint disable